src/vec2.rs

```diff
@@ -268,6 +272,7 @@ impl_vecn_scalar_bit_op_traits!(UVec2, u32, XYU32);
 
 impl_vecn_bit_op_traits!(UVec2, XYU32);
 
+#[cfg(not(feature = "cuda"))]
```

Comment:
I think it would be preferable to use `#[cfg]` blocks inside a common test instead of duplicating the tests. Primarily because which alignment is different, size should be the same. It looks like you've taken this approach in the `tests/*.rs` changes already.

Reply:
On second thought I see this is following the style of existing code, so ignore me. I'll make that change down the line.